drivers/hmis_external_apis/app/models/hmis_external_apis/ac_hmis/exporters/cde_export.rb

```diff
@@ -27,6 +30,53 @@ def run!
         write_row(values)
       end
 
+      # Include Unit Type assignment for all Enrollments at Walk-in projects
+      write_unit_occupancies
+      # Include Auto Exit flag for auto-exited Enrollments
+      write_auto_exits
+    end
+
+    private
+
+    def columns
+      [
+        'ResponseID',
+        'CustomFieldKey',
+        'RecordType',
+        'RecordId',
+        'Response',
+        'DateCreated',
+        'DateUpdated',
+      ]
+    end
+
+    def cdes
+      @cdes ||= Hmis::Hud::CustomDataElement.
+        where(data_source: data_source).
+        preload(:data_element_definition)
+    end
+
+    def walk_in_cded
+      @walk_in_cded ||= Hmis::Hud::CustomDataElementDefinition.where(data_source: data_source, key: :direct_entry, owner_type: 'Hmis::Hud::Project').first!
+    end
+
+    def write_auto_exits
+      # Include Auto Exit flag for auto-exited Enrollments
+      Hmis::Hud::Enrollment.hmis.auto_exited.preload(:exit).each do |enrollment|
+        values = [
+          enrollment.exit.id,  # ResponseID
+          AUTO_EXIT_KEY,       # CustomFieldKey
+          'Enrollment',        # RecordType
+          enrollment.id,       # RecordId
+          'true',              # Response
+          enrollment.exit.auto_exited,  # DateCreated (passing the date that the auto-exit occurred)
+          enrollment.exit.DateUpdated,  # DateUpdated
```

Comment:
A spec was not provided, but this should be sufficient. I did not include the actual `ExitDate` here because we don't easily have the Exit Date that was specified by the AutoExitJob. (Because the ExitDate could have changed after the auto-exit occurred).